src/opm/input/eclipse/share/keywords/900_OPM/G/GASINWAT

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "GASINWAT",
```

Comment:
Just a suggestion on my side. Wouldn't be better to call this keyword something like DISGASW? It would follow the convention we are using for other keywords like RSW, PVTGW, etc. In addition, the keyword would be following DISGAS in the manual, making it more intuitive and easier to the user find the feature.

Reply:
Good suggestion. My first implementation had DISGASWAT, but that didn't work due to more than 8 characters. But DISGASW is a good name. I will amend the PRs accordingly.